torch/serialization.py

```diff
@@ -735,7 +783,7 @@ def save(
     # documentation. We need it so that Sphinx doesn't leak `pickle`s path from
     # the build environment (e.g. `<module 'pickle' from '/leaked/path').
 
-    """save(obj, f, pickle_module=pickle, pickle_protocol=DEFAULT_PROTOCOL, _use_new_zipfile_serialization=True)
+    """save(obj, f, pickle_module=pickle, pickle_protocol=DEFAULT_PROTOCOL, _use_new_zipfile_serialization=True)  # noqa: B950
```

Comment:
check that it renders well

Reply:
Back to rendering ok :) https://docs-preview.pytorch.org/pytorch/pytorch/134504/generated/torch.save.html#torch-save